ur20_moveit_config/launch/planning_context.launch

```diff
@@ -0,0 +1,25 @@
+<launch>
+  <!-- By default we do not overwrite the URDF. Change the following to true to change the default behavior -->
+  <arg name="load_robot_description" default="false"/>
+
+  <!-- The name of the parameter under which the URDF is loaded -->
+  <arg name="robot_description" default="robot_description"/>
+
+  <!-- Load universal robot description format (URDF) -->
+  <param if="$(arg load_robot_description)" name="$(arg robot_description)" command="xacro  '$(find ur_description)/urdf/ur20.xacro'"/>
+
+  <!-- The semantic description that corresponds to the URDF -->
+  <param name="$(arg robot_description)_semantic" textfile="$(find ur20_moveit_config)/config/ur20.srdf" />
+
+  <!-- Load updated joint limits (override information from URDF) -->
+  <group ns="$(arg robot_description)_planning">
+    <rosparam command="load" file="$(find ur20_moveit_config)/config/joint_limits.yaml"/>
```

Comment:
I was having the same though, but since we do the same thing in the other packages I dropped those thoughts. Personally, I would be fine with postponing this to a follow-up PR where we change that in all MoveIt support packages.

Reply:
See #668